src/map.lua

```diff
@@ -29,13 +29,13 @@ function map.load(name)
     assert(#map.current.tilesets <= 1)
 
     -- initialize tilesets
-    for k, v in ipairs(map.current.tilesets) do
+    for _, v in ipairs(map.current.tilesets) do
```

Comment:
Can you quickly explain this change?

Is it that 'k' is an unused variable so we replace it with a '_' to indicate this?

Reply:
Yes, it's the preferred style for anonymous variables. It's then ignored by the linter as an unused.